src/richie/apps/core/templates/richie/pagination.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n %}{% spaceless %}
+
+{% if is_paginated %}
+<div class="pagination">
+  <div id="pagination-label" class="offscreen">{% trans "Pagination" %}</div>
+
+  <ul role="navigation" aria-labelledby="pagination-label" class="pagination__list">
+  {% for page in pages %}
+    {% if page %}
+      {% ifequal page page_obj.number %}
+        <li class="pagination__list__item pagination__list__item--current">
+          <span class="pagination__list__item__page-number">
+            <span class="offscreen">
+            {% if forloop.last %}
+              {% blocktrans %}Currently reading last page {{ page }}{% endblocktrans %}
+            {% else %}
+              {% blocktrans %}Currently reading page {{ page }}{% endblocktrans %}
+            {% endif %}
+            </span>
+            <span aria-hidden="true">{% if page == 1 %}Page {{ page }}{% else %}{{ page }}{% endif %}</span>
+          </span>
+        </li>
+      {% else %}
+        {% if page == 1 %}
+          <li class="pagination__list__item">
+            <a href="{{ request.path }}{% if getvars %}?{{ getvars|slice:"1:" }}{% endif %}" class="pagination__list__item__page-number">
+              <span class="offscreen">{% blocktrans %}Page {{ page }}{% endblocktrans %}</span>
```

Comment:
This is redundant: we can just keep one "Page 1" and remove the `offscreen` class and `aria-hidden` attribute

Reply:
This is not really redundant as first element has a special trick to ignore the "page=1" argument when building url such as it create something like `/mylist/` instead of `/mylist/?page=1` or  `/mylist/?sessionid=foo` instead of `/mylist/?page=1&sessionid=foo` if there was other argument. 

Also i don't understand the point about to remove offscreen class and the span since i reproduced the exact behavior from search pagination.

Reply:
> Also i don't understand the point about to remove offscreen class and the span since i reproduced the exact behavior from search pagination.

This is the entirety of what I was talking about.

The search pagination has a mistake here: if the content is exactly the same, it should exist only once, and not once with `aria-hidden` and once `offscreen`

Reply:
Ok, removed those two spans around text from first page item.